src/api_recievers/__tests__/influx_tests.js

```diff
@@ -0,0 +1,44 @@
+const { InfluxDB } = require('@influxdata/influxdb-client');
+
+const { AppData } = require('../influxqueries');
+
+
+jest.mock('@influxdata/influxdb-client', () => {
+  const mockWriteApi = {
+    writePoint: jest.fn(),
+    close: jest.fn()
+  };
+
+  const mockQueryApi = {
+    queryRows: jest.fn() 
+  };
+
+  return {
+    InfluxDB: jest.fn().mockImplementation(() => ({
+      getWriteApi: jest.fn().mockReturnValue(mockWriteApi),
+      getQueryApi: jest.fn().mockReturnValue(mockQueryApi) 
+    })),
+    Point: jest.fn().mockImplementation(() => ({
+      stringField: jest.fn().mockReturnThis(),
+      timestamp: jest.fn().mockReturnThis(),
+      tag: jest.fn().mockReturnThis()
+    })),
+    currentTime: {
+      seconds: jest.fn().mockReturnValue(Date.now() / 1000)
+    }
+  };s;
```

Comment:
This random s is causing one of the jest tests to fail

Reply:
It's gone now.